pages/en/lb3/Third-party-login-using-Passport.md

```diff
@@ -297,6 +297,28 @@ For more information, see [Configuring providers.json](Configuring-providers.js
 }
 ```
 
+### Configuration in model-config.json
+
+loopback-component-passport comes with ApplicationCredential, UserCredential and UserIdentity models. These need to be configured in model-config.json.
+
+Add "./node_modules/loopback-component-passport/lib/models" to the sources array.
+
```

Comment:
IIUC, you are saying to add the last line to the array as shown below:

```
"_meta": {
    "sources": [
      "loopback/common/models",
      "loopback/server/models",
      "../common/models",
      "./models", 
      "../node_modules/loopback-component-passport/lib/models"
    ],
```

If that's right, then I think we should say that in the doc to make it really clear.